src/open_inwoner/components/templates/components/StatusIndicator/StatusIndicator.html

```diff
@@ -0,0 +1,6 @@
+{% load icon_tags %}
```

Comment:
Made this into a component, in case we want to add more options besides `action_required`

Reply:
We do, the design has various levels/colors.

I'd say we use the options: info/warning/failure/success

Reply:
I added the remaining options